docs/tutorials/gromacs/spack-gromacs.md

```diff
@@ -0,0 +1,287 @@
+# HPC Toolkit - Install and Run Gromacs on a Slurm Cluster
+
+HPC Toolkit is an open-source software offered by Google Cloud which makes it
+easy for customers to deploy HPC environments on Google Cloud.
+
+In this tutorial you will use the HPC Toolkit to:
+
+* Deploy a [Slurm](https://github.com/SchedMD/slurm-gcp#readme) HPC cluster on
+  Google Cloud
+* Use [Spack](https://spack.io/) to install the Gromacs application and all of
+  its dependencies
+* Run a [Gromacs](https://www.gromacs.org/) job on your newly provisioned
+  cluster
+* Tear down the cluster
+
+## Select a Project
+
+Select a project in which to deploy an HPC cluster on Google.
+
+<walkthrough-project-setup billing="true"></walkthrough-project-setup>
+
+Once you have selected a project, click START.
+
+## Enable APIs & Permissions
+
+_Skip this step if you already ran this as part of a previous tutorial._
+
+In a new Google Cloud project there are several apis that must be enabled to
+deploy your HPC cluster. These will be caught when you perform `terraform apply`
+but you can save time by enabling them now by running:
+
+<!-- Tried the native way to do this and it timed out. Leaving comment here for future reference. -->
+<!-- <walkthrough-enable-apis apis="file.googleapis.com,compute.googleapis.com"></walkthrough-enable-apis> -->
+
+```bash
+gcloud services enable --project <walkthrough-project-id/> file.googleapis.com compute.googleapis.com
+```
+
+We also need to grant the default compute service account project edit access so
+the slurm controller can perform actions such as auto-scaling.
+
+<!-- Tried getting PROJECT_NUMBER using <walkthrough-project-number/> but returns empty string. -->
+
+```bash
+PROJECT_NUMBER=$(gcloud projects list --filter=<walkthrough-project-id/> --format='value(PROJECT_NUMBER)')
+
+echo "granting roles/editor to $PROJECT_NUMBER-compute@developer.gserviceaccount.com"
+
+gcloud iam service-accounts enable --project <walkthrough-project-id/> "$PROJECT_NUMBER"-compute@developer.gserviceaccount.com
+
+gcloud projects add-iam-policy-binding <walkthrough-project-id/> --member=serviceAccount:"$PROJECT_NUMBER"-compute@developer.gserviceaccount.com --role=roles/editor
+```
+
+## Build the Toolkit Binary
+
+*Skip this step if you already ran this as part of a previous tutorial.*
+
+To build HPC Toolkit binary from source run:
+
+```bash
+make
+```
+
+You should now have a binary named ghpc in the current directory. To verify the
+build run:
+
+```bash
+./ghpc --version
+```
+
+This should show you the version of the HPC Toolkit you are using.
```

Comment:
We could but I feel like it will be misleading as 1. they can only see 1.0.0 as this is where our branch originates and 2. these tutorials will never actually be available on 1.0.0 because by the time they are actually release it will be a higher version. 

Really the point of having them call version is just to make sure the binary built. I would vote to leave as it is.